src/js/components/Rating/Rating.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Rating = ({ title, rating, ratingType }) => {
+  const [ratingInt, ratingDecimal] = rating.toString().split('.');
+  const styleSuffix = rating === 'n/a' ? 'na' : Math.ceil(rating / 20) * 20;
+  const isMax = rating >= 100 || rating === 'n/a';
+  const barWidth = `${rating}%`;
+
+  return (
+    <div className={`rating rating--${ratingType}`}>
+      <div className="rating__number-border">
+        <div className={`rating__border--${styleSuffix} rating__number-wrap me-2 d-inline`}>
+          <span className={`rating__number-big user-select-none ${isMax ? 'rating__is-100' : ''}`}>{ratingInt}</span>
+          <span className="rating__number-small strong user-select-none">
+            {isMax || `.${ratingDecimal || '0'}`}
+          </span>
+        </div>
+      </div>
+      <span className="rating__title display-5 text-left mb-2 user-select-none" color="secondary">
+        {title}
+      </span>
```

Comment:
```suggestion
      <h3 className="rating__title display-5 text-left mb-2 user-select-none" color="secondary">
        {title}
      </h3>
```

It would help our SEO if we use HTML semantically https://www.thoughtco.com/why-use-semantic-html-3468271

Reply:
Yea makes sense... but are these titles?